lib/install.js

```diff
@@ -1017,12 +1017,19 @@ function showInstallGraph(pkg) {
     ui.log('info', '\nInstalled versions of %' + pkg.name + '%');
     visitForkRanges(installed, pkg, null, null, function(name, parent, resolved, range) {
       found = true;
-      if (range.version === '')
-        range.version = '*';
-      var rangeVersion = range.name === resolved.name ? range.version : range.exactName;
-      if (range.version === '*')
-        range.version = '';
-
+      var rangeVersion;
+      if (range) {
+        if (range.version === '') {
+          range.version = '*';
+        }
+        rangeVersion = range.name === resolved.name ? range.version : range.exactName;
+        if (range.version === '*') {
+          range.version = '';
+        }
+      }
+      else {
+        rangeVersion = resolved.version; 
+      }
       if (!parent)
         ui.log('info', '\n       %' + name + '% `' + resolved.version + '` (' + rangeVersion + ')');
```

Comment:
Perhaps just handle `rangeVersion` as `undefined` here meaning the brackets are just skipped or replaced with `(unspecified)` as you say.

Reply:
Roger